tools/osbuild-depsolve-dnf

```diff
@@ -204,7 +209,7 @@ class Solver():
         return packages
 
     def depsolve(self, transactions):
-        last_transaction = []
+        last_transaction: list = []
```

Comment:
This is incompatible with our supported Python version; it needs to be `typing.List`.

Reply:
Bah!  Thanks.  My bad.